usb/esp_modem_usb_dte/esp_modem_usb_api_target.cpp

```diff
@@ -1,11 +1,14 @@
 /*
- * SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 #include "usb_terminal.hpp"
 #include "cxx_include/esp_modem_api.hpp"
+#include "../private_include/exception_stub.hpp"
```

Comment:
Nitpick, could we add this to `PRIVATE_INCLUDE` paths?
and simply do:
```suggestion
#include "exception_stub.hpp"
```
(Just a personal preference, but don't very like relative `../` in includes, as they could be tricky -- for example can violate CMake canonical include path approach)

Reply:
This private include comes from `esp_modem` so I updated `usb/esp_modem_usb_dte/CMakeLists.txt` to include it, PTAL!